### ISSUE TITLE
Castle building window in Knight City has wrong flags on a picture

### ISSUE THREAD

**Issue by Branikolog** [ihhub/fheroes2#1633]
When we ant to build a castle in Knight city, we press on a tent, and in the window appeared, we can see a picture of castle with red flags. When we build castle - it has blue.
The same red flags we can see even in the original game. 
![image](https://user-images.githubusercontent.com/55348946/92932789-0bc04d80-f44e-11ea-940c-584c6efe03cd.png)

**Comment by idshibanov** [comment]
Beta Knight castle screen had red flags (possibly reused from Heroes 1). Upgraded units were supposed to be red in early game development I believe.

**Comment by idshibanov** [comment]
Can't call this a "bug" by the way, more like artist oversight. @ihhub I think we need a label like "assets" for cases like this one where we need to create a custom sprite.

**Comment by Branikolog** [comment]
> Beta Knight castle screen had red flags (possibly reused from Heroes 1). Upgraded units were supposed to be red in early game development I believe.

Yes, that's true. I saw original beta screenshot of knight castle with red flags...

**Comment by ihhub** [comment]
The problem using blue flag would be the background which is blue again so the original developers might had their reason not to use blue flag there.

**Comment by ihhub** [comment]
This is how it would look like this blue flag:
![castle_resized](https://user-images.githubusercontent.com/19829520/115989435-ee14b280-a5f0-11eb-9be8-bf33a2fbe1d5.png)

**Comment by ihhub** [comment]
Let's vote then (@idshibanov , @oleg-derevenetz , @Branikolog  and @LeHerosInconnu ):
Option **1**: we keep as it is and close this issue
Option **2**: we're going to regenerate (or just replace red color with blue)

**Comment by oleg-derevenetz** [comment]
Hi all, I vote for the option to keep it as is.

**Comment by Branikolog** [comment]
Hi.
I'd like to watch exactly the same picture of a building I'm going to purchase.
Vote for option 2.

**Comment by LeHerosInconnu** [comment]
Hello all,
I would prefer the color to be blue.
I vote for option 2.
